Apps/phvxstream/README.md

```diff
@@ -0,0 +1,40 @@
+# VxPhantom
+
+The Falcon Sandbox Phantom App
+
+Licensed under Apache 2.0 (https://www.apache.org/licenses/LICENSE-2.0.txt)
+
+Copyright (C) 2018 Hybrid Analysis GmbH
```

Comment:
@ricardo-phantom 

Since there is no `README.md` at https://github.com/ricardo-phantom/phantom-apps/tree/next/Apps/phdigitalshadows, Could you check if it's okay to keep this line `Copyright (C) 2018 Hybrid Analysis GmbH` here?

Reply:
Yes looks good